client/app/components/elements/Logo/Logo.js

```diff
@@ -5,10 +5,15 @@ import { className } from '../../../utils/PropTypes';
 
 import css from './Logo.css';
 
-const logoContent = function logoContent(image, text) {
+const logoContent = function logoContent(image, imageHighRes, text) {
   let content = null;
   if (image) {
```

Comment:
This only checks that `image` exists, but expects also `imageHighRes` as well. Maybe add that to the condition?